openapi/paths/aml-checks@{id}@stop-review.yaml

```
@@ -0,0 +1,34 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+post:
+  x-products:
+    - Core
+  tags:
+    - AML
+  summary: Stop review of an AML check
+  operationId: PostAmlCheckStopReview
+  x-sdk-operation-name: stopAmlCheckReview
+  description: |-
+    Stops the review of an AML check with a specified ID.
```

suggestion: I'm not sure if this is for manual review (suggestion is based on starting a manual review)

```suggestion
    Stops the manual review process for an AML check with a specified ID.
```

---

Thanks @noon-dawg, I applied all your suggestions